### DISCUSSION TITLE
Panel not shown (installation on top of Jetstream)

### DISCUSSION THREAD

**Discussion by wq9578** [filamentphp/filament#9086] [Help]
On top of a running Jetstream Laravel app (with Livewire), I have installed Filament v3 with:
```
php artisan filament:install –panels
```

However, no panel shows up at `/admin` (or `/app` after creating another panel with `php artisan make:filament-panel app`).

The service providers are registered correctly in the `providers` array in `config/app.php`:

```
        App\Providers\Filament\AdminPanelProvider::class,
        App\Providers\Filament\AppPanelProvider::class,
```

What can I do?

**Comment by leandrocfe** [comment]
This was supposed to work.
Did you run `composer require filament/filament`, right?

What about `php artisan filament:upgrade` ?

Could you share the project on github please?

**Comment by wq9578** [comment]
Yes, I ran `composer require filament/filament:"^3.0-stable" -W` as described in the [docs](https://filamentphp.com/docs/3.x/panels/installation).

I also tried `php artisan filament:upgrade` now, but the problem still persists.

I cannot share the project as a whole, but can answer questions.

**Reply by leandrocfe** [reply]
Could you create another project with the setup jetsteam + filament and share with us, without data or resources, just to check if it is working?

**Reply by wq9578** [reply]
With a fresh installation, it works: https://github.com/wq9578/jetsteam-filament

**Reply by leandrocfe** [reply]
Yes, some config in your project... What about your AppServiceProvider?

**Reply by wq9578** [reply]
If I use the file from the fresh installation (https://github.com/wq9578/jetsteam-filament/blob/main/jetstream_filament_app_test/app/Providers/AppServiceProvider.php) nothing changes.

**Reply by leandrocfe** [reply]
Probably a setup issue but I can't help you without the code

**Reply by wq9578** [reply]
Maybe I should simply start a new Laravel app and copy the logic to the new one.
Thank you very much so far!

**Reply by leandrocfe** [reply]
yes, this should work
